### PR TITLE
You can order pizza from eda.by

### PR DESCRIPTION
**Pull request by chyrta**

### CODE REVIEW

File: scripts/edaby.coffee [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+  Author:
+    Dzmitry Chyrta
+###
+
+URL = 'http://eda.by/api.php'
+CLIENT_NAME = 'Х'
+CLIENT_EMAIL = 'Х'
+CLIENT_PHONE = '+375+(ХХ)+ХХХ-ХХ-ХХ'
+CLIENT_ADDRESS = 'Х'
```
Comment (dzhlobo) [inline scripts/edaby.coffee]: Let's use environment variables instead of constants. So it would be easier to configure it on server.

**Comment by dzhlobo** [comment]
@chyrta, cool, thank you.

Is it possible to choose size of pizza? On eda.by I see 32cm and 42cm. Sometimes I'd like to order the big one. It's not necessary to be in this PR.

**Comment by DDKatch** [comment]
@chyrta do you think this PR is still actual? I'm curious because we're going to migrate at `botkit` and it may cause incompatibility of this code (high probability) with the whole app.

**Comment by AleksSenkou** [comment]
Going to move it to an issue saying that food can be requested for meetup via talking with Mona.
Unfortunately, we can't merge and support it right now because moving to a new engine for Mona

**Pull request closed without merging**